manifests/package.pp

```diff
@@ -106,4 +112,16 @@
     require => Package[$container_package],
   }
 
+  if $experimental_filters == true {
+    package { $repose::params::experimental_filters_packages:
+      ensure => $package_ensure, 
+      require => Package[$container_package],
+    }
+  } else {
+    package { $repose::params::experimental_filters_packages:
+      ensure => absent, 
```

Comment:
Updated major version as requested in conversation with @teriyakichild  to retain the logic here. People upgrading the repose module should be aware of the change to manage the experimental filters package w/in the module.